examples/hp_search_benchmarks/darts_penntreebank_pytorch/model_def.py

```diff
@@ -118,49 +149,6 @@ def get_genotype_from_hps(self):
             cell_config.append((edge_op, edge_ind))
         return Genotype(recurrent=cell_config, concat=range(1, 9))
 
-    def build_model(self) -> nn.Module:
```

Comment:
Can we frontload these changes into the next release?  I didn't even realize that we still had old-style expamples laying around.